app/scripts/controllers/metametrics.test.js

```diff
@@ -561,6 +561,80 @@ describe('MetaMetricsController', function () {
     });
   });
 
+  describe('Change Transaction XXX anonymous event namnes', function () {
+    it('should change "Transaction Added" anonymous event names to "Transaction Added Anon"', function () {
+      const metaMetricsController = getMetaMetricsController();
+      const spy = sinon.spy(segment, 'track');
```

Comment:
Can you check if it's failing due to this? https://github.com/MetaMask/metamask-extension/actions/runs/6178765858/job/16772582023?pr=20736

Reply:
Yeah, That's weird because this exact same code is used elsewhere in this file before now...I just copied and used the code.

Reply:
Spoke to Dan and this error doesn't stop merging. The extension team is working on making the CI step required for only new files.